plugins/system/debug/src/DataCollector/MemoryCollector.php

```diff
@@ -0,0 +1,153 @@
+<?php
+/**
+ * This file is part of the DebugBar package.
+ *
+ * @copyright  (c) 2013 Maxime Bouroumeau-Fuseau
+ * @license    For the full copyright and license information, please view the LICENSE
+ *             file that was distributed with this source code.
+ */
+
+namespace Joomla\Plugin\System\Debug\DataCollector;
+
+use Joomla\Plugin\System\Debug\AbstractDataCollector;
+use Joomla\Registry\Registry;
+
+/**
+ * Collects info about the request duration as well as providing
+ * a way to log duration of any operations
+ *
+ * @since  __DEPLOY_VERSION__
+ */
+class MemoryCollector extends AbstractDataCollector
+{
+	/**
+	 * @var boolean
+	 *
+	 * @since __DEPLOY_VERSION__
+	 */
+	protected $realUsage = false;
+
+	/**
+	 * @var float
+	 *
+	 * @since __DEPLOY_VERSION__
+	 */
+	protected $peakUsage = 0;
+
+	/**
+	 * Constructor.
+	 *
+	 * @param Registry    $params Parameters.
+	 * @param float       $peakUsage
+	 * @param boolean     $realUsage
```

Comment:
Spaces as per https://developer.joomla.org/coding-standards/docblocks.html

```
variable type (There must be 3 spaces before variable type.)
variable name (There must be 2 spaces after the longest type.)
variable description (There must be 2 spaces after the longest variable name.)
```


```suggestion
	 * @param   Registry  $params Parameters.
	 * @param   float     $peakUsage
	 * @param   boolean   $realUsage
```

Reply:
@PhilETaylor Thanks, it's always a pain, I set the PHPStorm code style "From Joomla" and it looks like either outdated or incorrect.